### ISSUE TITLE
Preserve Line Numbers

### ISSUE THREAD

**Issue by tkjone** [tkjone/pretty-logs#22]
Right now when we log it is going to always show the source code line number of this lib, but we want to show the line number in the developers source code.

**Comment by skidding** [comment]
Can't think of a 3rd solution. Unfortunately I don't have experience with this. Personally I'm OK with dropping the heavy formatting.

**Comment by tkjone** [comment]
Yeah, does seem like the quest for beauty got in the way of it actually doing what it was supposed to do.  Thanks for the input!
